### PR TITLE
Fix Drupal coding standards in Drush aliases template

### PR DESCRIPTION
**Pull request by christopher-hopper**
## Proposed Changes

- Add missing file doc comment
- Add full-stops to end of inline comments
- Add comma following last multiline array items

## Related Issues

- #37

### CODE REVIEW

File: Resources/templates/aliases.php.twig [hunk 1]
```diff
@@ -1,13 +1,20 @@
 <?php
 
+{% for alias in aliases|slice(0,1) %}
+/**
+ * @file
+ * Drush site-alias configuration for {{ aliases["ac-site"] }}.
```
Comment (christopher-hopper) [inline Resources/templates/aliases.php.twig]: **Question**  
Will this key `ac-site` always be available, or should we check for either `ac-site` or `site` depending on whether its Acquia Site Factory or Acquia Cloud?

Comment (kylebrowning) [inline Resources/templates/aliases.php.twig]: Its only available for ACE, https://github.com/acquia/club/blob/master/src/Command/ACAliasesCommand.php#L119
And site is only avialable for for ACSF
https://github.com/acquia/club/blob/master/src/Command/ACAliasesCommand.php#L155

That being said we could fix that.

Comment (christopher-hopper) [inline Resources/templates/aliases.php.twig]: Perhaps for now I could just simplify this comment and leave out the `ac-site` or `site` keys. Identifying it in the `@file` doc comment is a _nice to have_, but not strictly necessary.

**Comment by kylebrowning** [comment]
I have no forgotten about this, just need some time to wrap up other things. Expect this to be merged end of week after I test it.

**Pull request merged**